umap/static/umap/js/modules/schema.js

```diff
@@ -313,6 +313,10 @@ export const SCHEMA = {
     impacts: ['ui', 'data'],
     label: translate('name'),
   },
+  numberOfConnectedPeers: {
```

Comment:
I'm not convinced this has to be in the schema here, given it's not a stored property. I'd simply have a method that updates this counter (with a css class accessor or something), wdyt ?

Reply:
I was wondering the same thing when writing it. Using the schema is mainly a way to tell what part of the interface needs to be updated. If we weren't to use the schema, we would special-case this in the `render()` method of the map.

I don't think there is a clear winner between the two ways here, I believe we just have to decide on one and stick to it for the other cases as well. On one case we have to check not to forward these events to other peers, and on the other we have to call the part that updates the interface after an if branch testing for this specific property.

Reply:
I'll let you decide @yohanboniface 👍

Reply:
I'm trying to get the full picture so to have an opinion on this.
Two elements:
- I'm expecting the schema to reflect the actual data, so if we add `numberOfConnectedPeers` to the schema, it means it's somewhere on the map object (I guess in `options` like others for now)
- maybe we can keep this property in the schema, and add a `sync` property to schema entries, defaulting to true, but when false we do not sync changes to other peers ?